.github/workflows/lint.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php: ['7.2', '7.3', '7.4']
+        php: ['7.4', '8.0', '8.1']
```

Comment:
Note to self: Check the previously closed PRs for other updates (checkout v3 etc.)